src/MandrillTransport.ts

```diff
@@ -1,5 +1,7 @@
-import type {Transport, SentMessageInfo} from 'nodemailer';
+import type {Transport} from 'nodemailer';
 import type MailMessage from 'nodemailer/lib/mailer/mail-message';
+import MimeNode from 'nodemailer/lib/mime-node';
```

Comment:
**suggestion**: use import type and import only type you need
`import type {Envelope} from 'nodemailer/lib/mime-node';`

Reply:
done